### PR TITLE
Standardization of formatting on Plugin pages.

### PR DESCRIPTION
**Pull request by yb-f**
Removed plugins that no longer existed and attempted to standardize formatting across plugin pages to make a more readable and pleasurable experience.

Not all of the pages for plugins have been standardized yet, but it is a start.

### CODE REVIEW

File: plugins/community-plugins/mq2aapurchase.md [hunk 1]
```diff
@@ -14,25 +14,25 @@ You can find the latest version of MQ2AAPurchase [here](https://macroquest.org/p
 
 ## Commands
 
-`/aapurchase help -- List commands, aka this`  
-`/aapurchase now -- start buying now`  
-`/aapurchase add \"AA Name\" rank -- M for max instead of a rank`  
-`/aapurchase bank # -- set the amount of points to bank`  
-`/aapurchase load -- Reload INI`
+`/aapurchase help` -- List commands, aka this<br>
```
Comment (brainiac) [inline plugins/community-plugins/mq2aapurchase.md:17]: I would look at using the definition list format like we use for datatypes/tlos, or using a list, or maybe even a table. Using inline code blocks and html tags like this is an indicator that you're going against the grain.

File: plugins/community-plugins/mq2advpath.md [hunk 2]
```diff
@@ -6,30 +6,31 @@ This plugin allows you to record and playback player movement.
...
+| [_string_](../../reference/data-types/datatype-string.md) | **X[Check Point Name OR Waypoint number]** | LOC |
+| [_string_](../../reference/data-types/datatype-string.md) | **Z[Check Point Name OR Waypoint number]** | LOC |
+| [_spawn_](../../reference/data-types/datatype-spawn.md) | **Monitor**      | Spawn you're following |
+| [_int_](../../reference/data-types/datatype-int.md) | **Idle**         | Idle time when following and not moving |
+| [_float_](../../reference/data-types/datatype-float.md) | **Length**       | Estimated length of the follow path |
+| [_bool_](../../reference/data-types/datatype-bool.md) | **Following**    | Following spawn |
+| [_bool_](../../reference/data-types/datatype-bool.md) | **Playing**      | Playing path |
+| [_bool_](../../reference/data-types/datatype-bool.md) | **Recording**    | Recording path |
+| [_int_](../../reference/data-types/datatype-int.md) | **Status**       | Status 0 = off , 1 = on , 2 = paused |
+| [_bool_](../../reference/data-types/datatype-bool.md) | **Paused**       | Paused |
```
Comment (brainiac) [inline plugins/community-plugins/mq2advpath.md]: Look at how we're linking in the datatype pages with reference links to keep this easier to maintain.

**Review by brainiac — CHANGES_REQUESTED** [review]
Left two comments but they apply to the whole thing.

**Comment by yb-f** [comment]
Will work on doing that tomorrow. Thanks for the input!

**Comment by brainiac** [comment]
close enough.

**Pull request merged**